Kudu.Core/Jobs/JobLogger.cs

```diff
@@ -68,6 +68,20 @@ protected string GetStatusFilePath()
         public void ReportStatus<TJobStatus>(TJobStatus status) where TJobStatus : class, IJobStatus
         {
             ReportStatus(status, logStatus: true);
+            if (status.GetType() == typeof(ContinuousJobStatus))
```

Comment:
Maybe a comment explaining why we have to clear the cache after we report the status?